h2o-algos/src/test/java/hex/generic/GenericModelTest.java

```diff
@@ -528,26 +529,25 @@ public void downloadable_mojo_gbm() throws IOException {
             parms._distribution = AUTO;
             parms._response_column = trainingFrame._names[1];
             parms._ntrees = 1;
+            parms._seed = 0XFEED;
```

Comment:
@Pscheidl do we need this in all downloadable tests? or we can remove it here?

Reply:
We can remove it. It is actually better to produce a variety of models. I'll do it.